### PR TITLE
separate null values from coin_distributions_by_earner to reviewed

### PR DESCRIPTION
**Pull request by ice-cronus**

### CODE REVIEW

File: coin-distribution/DDL.sql [hunk 1]
```diff
@@ -83,21 +83,22 @@ CREATE INDEX IF NOT EXISTS coin_distributions_pending_review_lookup3_ix ON coin_
 CREATE INDEX IF NOT EXISTS coin_distributions_pending_review_lookup4_ix ON coin_distributions_pending_review (ice,username,internal_id);
 CREATE INDEX IF NOT EXISTS coin_distributions_pending_review_lookup5_ix ON coin_distributions_pending_review (referred_by_username,internal_id);
 CREATE INDEX IF NOT EXISTS coin_distributions_pending_review_lookup6_ix ON coin_distributions_pending_review (ice,referred_by_username,internal_id);
+CREATE INDEX IF NOT EXISTS coin_distributions_pending_review_nulls_ix   ON coin_distributions_pending_review (internal_id NULLS FIRST);
 
 CREATE TABLE IF NOT EXISTS reviewed_coin_distributions  (
                     reviewed_at               timestamp NOT NULL,
-                    created_at                timestamp NOT NULL,
-                    internal_id               bigint    NOT NULL,
```
Comment (ice-ares) [inline coin-distribution/DDL.sql:90]: Nope, u dont touch this table

**Review by ice-ares — CHANGES_REQUESTED** [review]

File: coin-distribution/DDL.sql (hunk 1 above)
Comment (ice-cronus) [reply coin-distribution/DDL.sql:90]: But do we need to store that entries as reviewed?

Comment (ice-ares) [reply coin-distribution/DDL.sql:90]: Yes, but coalesce it

**Comment by ice-ares** [comment]
I've added em myself

**Pull request closed without merging**